src/stock-operations/batch-no-selector/batch-no-selector.component.tsx

```diff
@@ -102,7 +102,7 @@ const BatchNoSelector = <T,>(props: BatchNoSelectorProps<T>) => {
             }}
             initialSelectedItem={initialSelectedItem}
             itemToString={(s: StockBatchDTO) =>
-              s?.batchNo ? `${s?.batchNo}` : ""
+              s?.batchNo ? `${s?.batchNo} | Qty: ${s?.quantity ?? ""}` : ""
```

Comment:
Should we adopt some text in place of the ""? Empty string can be interpreted differently by different people and I feel we should be clear that we don't know the quantity in the batch. What do others think? @jabahum @makombe @slubwama

Reply:
I think there was a fix wea we dont even have to display the batch wen the quantity is 0. so i think just rendering the quantity would just be enough

Reply:
Can confirm to this if it still exists then move it in

Reply:
@jabahum I don't understand what you mean